_episodes/03-dicts.md

```diff
@@ -216,5 +242,32 @@ Green tea contains 28 mg of caffeine per serving
 > {: .solution}
 {: .challenge}
 
-[mayo-caffeine]: https://www.mayoclinic.org/healthy-lifestyle/nutrition-and-healthy-eating/in-depth/caffeine/art-20049372
+## Nested dicts
+
+It is worth noting that the values in a `dict` can be of any type
+(this is not true for the keys).
+One notable case, is that values can be themselves `dict`s:
+~~~
+nutrition_values = {'energy': {'units': 'kCal/100g',
+                               'values': {'white bread': 273,
+                                          'almonds': 512}},
+                    'caffeine': {'units': 'mg per serving',
+                                 'values': caffeine_mg_per_serving}}
+~~~
+{: .language-python}
+It is then possible to access data using multiple square bracket expressions:
+~~~
+print("Caffeine content of coffee:", nutrition_values['caffeine']['values']['coffee'])
+print("Units:", nutrition_values['caffeine']['units'])
+~~~
+{: .language-python}
+~~~
+Caffeine content of coffee: 96 
+Units: mg per serving
+~~~
+{: .output}
+
+
+
+[MAYO bk-caffeine]: https://www.mayoclinic.org/healthy-lifestyle/nutrition-and-healthy-eating/in-depth/caffeine/art-20049372
```

Comment:
I think this is just autocorrect or something spuriously changing this, but this link now appears to be broken (as the anchor above still points at `[mayo-caffeine]`). Was there a purpose to the change on this line or was it just a tool-assisted typo?